cmd/juju/commands/bootstrap.go

```diff
@@ -284,7 +284,7 @@ func (c *bootstrapCommand) configDetails() map[string]interface{} {
 	if modelCgf, err := cmdmodel.ConfigDetails(); err == nil {
 		addAll(modelCgf)
 	}
-	if controllerCgf, err := cmdcontroller.ConfigDetails(); err == nil {
+	if controllerCgf, err := cmdcontroller.ConfigDetailsAll(); err == nil {
```

Comment:
This is a drive-by fix so that `juju bootstrap -h` shows all the config keys, not just the updatable ones (and `juju controller-config -h` still shows only the updatable ones).